db/app.py

```diff
@@ -0,0 +1,93 @@
+from weakref import WeakKeyDictionary
+import MySQLdb
+import pandas as pd
+host = "localhost"
+user = "root"
+password = '78694142@Fj'
+db = 'kanto'
+port = 3306
+con = MySQLdb.connect(host,user,password,db,port)#linhas 1 a 8 estão pegando dados e usando para conectar ao db
```

Comment:
Always put comments on the line above. Also, they are very rarely necessary, and here definitely not.